src/utils/helpers/list/buildQueryFilters.ts

```diff
@@ -67,8 +105,13 @@ const constructFilter = (
   filterValue: string
 ) => {
   const filter = filterTypeDefinitions[type](filterValue, options)
+
   // If all of the criteria is undefined skip filter
-  if (!Object.values(filter).find((value) => value !== undefined)) return {}
+  // if (!Object.values(filter).find((value) => value !== undefined)) return {}
```

Comment:
Maybe remove

Reply:
Yeah, I was going to, but slightly concerned it might be needed for something in the ApplicationList. I haven't been able to see *what*, but I'm wary to remove it completely just in case. 

Maybe I'll remove it when we've had the filters feature branch functioning and well tested (along with the applications list). I'll do another overall tidy-up before merging the whole `feature/data-view-filters` into `develop`.